src/Symfony/Component/Validator/Tests/Fixtures/Attribute/Nested/Entity.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Fixtures\Attribute\Nested;
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Tests\Fixtures\Attribute\EntityParent;
+use Symfony\Component\Validator\Tests\Fixtures\EntityInterfaceB;
+use Symfony\Component\Validator\Tests\Fixtures\CallbackClass;
+use Symfony\Component\Validator\Tests\Fixtures\ConstraintA;
+
+#[
+    ConstraintA,
+    Assert\GroupSequence(['Foo', 'Entity']),
+    Assert\Callback([CallbackClass::class, 'callback']),
+]
+class Entity extends EntityParent implements EntityInterfaceB
+{
+    #[
+        Assert\NotNull,
+        Assert\Range(min: 3),
+        Assert\All([
+            new Assert\NotNull(),
+            new Assert\Range(min: 3),
+        ]),
+        Assert\All([
+            'constraints' => [
+                new Assert\NotNull(),
+                new Assert\Range(min: 3),
+            ],
+        ]),
+        Assert\Collection([
+            'fields' => [
+                'foo' => [
+                    new Assert\NotNull(),
+                    new Assert\Range(min: 3),
+                ],
+                'bar' => new Assert\Range(min: 5),
+            ]
+        ]),
+        Assert\Choice(choices: ['A', 'B'], message: 'Must be one of %choices%')
+    ]
+    public $firstName;
```

Comment:
the Ci reports `PHP Fatal error:  Constant expression contains invalid operations`